Dockerfile.win10.min

```diff
@@ -66,10 +66,7 @@ RUN vcpkg.exe update
 RUN vcpkg.exe install openssl:x64-windows openssl-windows:x64-windows rapidjson:x64-windows re2:x64-windows boost-interprocess:x64-windows boost-stacktrace:x64-windows zlib:x64-windows pthread:x64-windows b64:x64-windows
 RUN vcpkg.exe integrate install
 
-# Install a recent version of cmake. The version installed with VS is
```

Comment:
We may want to keep this comment. Unless there is a reason to delete it.

Reply:
I don't believe we need that comment anymore: 
1. ONNXRUNTIME requires 3.24.1 version as minimal as of 1.14.1 release 
2. Comment is irrelevant to me, CMAKE and VSCode are different frameworks where CMAKE actually is using VSCode compiler to build targets according to generated instructions.